csaf_2.0/submit/README.md

```diff
@@ -0,0 +1,84 @@
+# Submission request to advance CSAF v2.0 to an International Standard
+
+Date: January 28, 2023
+
+>*The following text parts - marked up in a gray, cursive,, and smaller font - have been copied directly from*
+>*the requirements stated in the relevant section of the current OASIS Liaison Policies page at:*
+>                                          https://www.oasis-open.org/policies-guidelines/liaison/#submitwork
+
+*Any submission request delivered to the OASIS President under this policy must be in writing, and must include the following:*
+
+a. *The name(s) of the submission requester(s), that is, the TC, the Member Section, or the OASIS*
+   *Organizational Members that support the submission request as described in section 1(d).*
+
+OASIS Common Security Advisory Framework (CSAF) TC
+
+b. *The name of the intended receiving standards organization. The request may also suggest the committee*
+   *or group in that organization which should process that submission.*
+
+ISO/IEC\ JTC\ 1 Secretariat (suggesting “SC\ 27 Information security, cybersecurity and privacy
+protection” as the published scope at https://www.iso.org/committee/45306.html indicates)
+
+c. *The intended status or outcome that the request seeks from the receiving organization’s process; and a*
+   *short description of the receiving organization’s approval process, including estimated time required,*
+   *stages of approval and who votes at each stage.*
+
+Intended status: Advance CSAF\ v2.0 OASIS Standard to an ISO/IEC International Standard
+
+Process: ISO/IEC\ JTC\ 1 PAS Transposition Process involving a Draft International
+Standard\(DIS) ballot followed by a Ballot Resolution Meeting\ (BRM) (if there are comments
+during ballots), and potentially a Final DIS\ (FDIS) Ballot.
+
+Stages of approval: https://www.iso.org/stage-codes.html - Stage 40
+
+
+*Note: Submission request continued on next page*
+
+                                              *... continued from first page of submission request.*
+
+d. *An explanation of how the submission will benefit OASIS.*
+
+Advancing CSAF\ v2.0 to an international standard draws international attention to the work
+done by OASIS and to the organization itself,and governments/regulators may be more
+inclined to use the standard because of its ISO/IEC status.
```

Comment:
```suggestion
Advancing CSAF v2.0 to an international standard draws international attention to the work
done by OASIS and to the organization itself, and governments/regulators may be more
inclined to use the standard because of its ISO/IEC status.

Reply:
Please cf. to the comment on space protection.